VS: district scraper fails

The [district scraper of VS currently fails](https://github.com/openZH/covid_19/runs/1353568829?check_suite_focus=true):

```
Populating database from CSV fallzahlen_kanton_VS_bezirk...
Run the district scraper...
Traceback (most recent call last):
  File "/home/runner/work/covid_19/covid_19/scrapers/scrape_vs_districts.py", line 97, in <module>
    district_values.append(int(value))
ValueError: invalid literal for int() with base 10: '146         16'
Error: Process completed with exit code 1.
```

--- a/scrapers/scrape_vs_districts.py
+++ b/scrapers/scrape_vs_districts.py
@@ -23,7 +23,7 @@ year = sc.find(r'\d+\.\d+\.(\d{4})', content)
 
 # last page contains the district data
 page = int(sc.pdfinfo(pdf))
-content = sc.pdftotext(pdf, page=page, layout=True, rect=[0, 403, 450, 50], fixed=2)
+content = sc.pdftotext(pdf, page=page, layout=True, rect=[0, 403, 420, 50], fixed=2)
 
 # strip everything including the "Anzahl Faelle" column + values
 def strip_left_number(content):
@@ -47,7 +47,7 @@ def strip_right_items(content):
     lines = content.split('\n')
     pos = None
     for line in lines:
-        res = re.search(r'(\d+\.\d+)\s?$', line)
+        res = re.search(r'(\d+|\d+\.\d+)\s?$', line)
         if res is not None:
             if pos is None:
                 pos = res.start()

